pkg/devicemapper/log.go

```diff
@@ -1,11 +1,80 @@
 package devicemapper
 
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
 // definitions from lvm2 lib/log/log.h
 const (
-	LogLevelFatal  = 2 + iota // _LOG_FATAL
-	LogLevelErr               // _LOG_ERR
-	LogLevelWarn              // _LOG_WARN
-	LogLevelNotice            // _LOG_NOTICE
-	LogLevelInfo              // _LOG_INFO
-	LogLevelDebug             // _LOG_DEBUG
+	LogLevelFatal  LogLevel = 2 + iota // _LOG_FATAL
+	LogLevelErr                        // _LOG_ERR
+	LogLevelWarn                       // _LOG_WARN
+	LogLevelNotice                     // _LOG_NOTICE
+	LogLevelInfo                       // _LOG_INFO
+	LogLevelDebug                      // _LOG_DEBUG
 )
+
+// Working "notice" level into mappings as that is not a direct mapping for logrus log levels.
+var logFunc = []func(format string, args ...interface{}){
+	logrus.Infof,
```

Comment:
There shouldn't be a need to pull in logrus into this package.
Instead we can do the conversion in the devicmapper graphdriver package to devmapper levels defined in this package.